File: project/server/src/main/java/ru/zagrebin/dto/PostCardDto.java
package ru.zagrebin.dto;

import lombok.Data;
import java.util.Set;

@Data
public class PostCardDto {
    private Long id;
    private String clientId;
    private String title;
    private String excerpt;
    private String coverUrl;
    private Long authorId;
    private String postType;
    private int likesCount;
    private Integer cookingTimeMinutes;
    private Integer calories;
    private String authorName;
    private String authorAvatarUrl;
    private String publishedAt;
    private Set<String> tags;
    private Long viewsCount;
    private boolean liked;
}

File: project/mobile/app/src/main/java/ru/zagrebin/culinaryblog/data/remote/dto/PostCardDto.kt
package ru.zagrebin.culinaryblog.data.remote.dto;

data class PostCardDto(
        val id: Long,
        val title: String,
        val excerpt: String,
        val coverUrl: String?,
        val authorId: Long?,
        val likesCount: Int,
        val cookingTimeMinutes: Int?,
        val calories: Int?,

        val authorName: String?,
        val publishedAt: String?,
        val tags: Set<String>?,
        val viewsCount: Long?
)
